### ISSUE TITLE
💡 [REQUEST] - Rip without download

### ISSUE THREAD

**Issue by gabrielbryk** [PlexRipper/PlexRipper#264]
### Checks

- [X] I have searched the existing issue and my feature request does not exist!
- [X] I have tried out the latest development build and can confirm my feature request is not yet implemented!

### Describe the feature

It would be awesome if a sonarr/radarr integration was added so I could rip the movies from shared plex servers without actually ripping the files. I want this because many of the shared servers I have access to don't have the bandwidth to feasibly download a large amount from and/or the format they have isn't what I prefer (too low quality or too high quality). 

The reason I think this would be a great feature is because my issue is very rarely finding a good torrent for a movie, but finding a good movie in the first place. 

### Additional information

- [X] Would you be willing to help implement this feature?
- [ ] Does this feature have a potential to break existing features?

**Comment by JasonLandbridge** [comment]
Thanks for the suggestion! So you mean that PlexRipper would send movies/tv-shows to sonarr/radarr which you can then download through sonarr/radarr?

**Comment by gabrielbryk** [comment]
Exactly, that way you aren't reliant on the upload speed of someones plex server and you can select your own quality profiles. This can also be implemented using the list functionality of sonarr/radarr and one of the websites that you can automate list creation like listrr.

**Comment by JasonLandbridge** [comment]
I did some quick research but it's not too clear, can you provide some more links and more detail?

I might have something similar here: https://github.com/PlexRipper/PlexRipper/issues/105

**Comment by gabrielbryk** [comment]
The mentioned issue is similar, I think what im suggesting achieves the same thing but with a different approach. How I see this working is that PlexRipper can rip all movie/show IDs (probably TMDB) from server and then both sonarr and radarr have pretty easy apis to add content using that info.

**Comment by JasonLandbridge** [comment]
I see, yeah that should be possible! I will add it as a todo

**Comment by aniel300** [comment]
> I see, yeah that should be possible! I will add it as a todo

any update on this?

**Comment by aniel300** [comment]
would it be possible to integrate this with the arrs as a indexer, to allow upgrades, etc?

**Comment by JasonLandbridge** [comment]
@aniel300 The discussion for this is taking place here: https://github.com/PlexRipper/PlexRipper/discussions/400

**Comment by aniel300** [comment]
@JasonLandbridge thank u
